cherab/tools/inversions/voxels.pyx

```diff
@@ -49,7 +50,8 @@ class AxisymmetricVoxel(Voxel):
 
     # cdef np.ndarray _vertices, _triangles
 
-    def __init__(self, vertices, parent=None, material=None):
+    def __init__(self, vertices, parent=None, material=None,
+                 primitive_type: str = 'mesh'):
```

Comment:
We haven't been type hinting in pure python classes and code. Although this is harmless, it does break against the style conventions used everywhere else in CHERAB core. When we add a doc string we can suggest the type there instead.

Reply:
I've now removed the type hinting.